src/main/java/io/jenkins/plugins/analysis/core/steps/RecordIssuesStep.java

```diff
@@ -837,6 +838,35 @@ public String getReferenceJobName() {
         return referenceJobName;
     }
 
+    /**
+     * Sets the reference build id of the reference job for the issue difference computation.
+     *
+     * @param referenceBuildId
+     *         the build id of the reference job
+     */
+    @DataBoundSetter
+    public void setReferenceBuildId(final String referenceBuildId) {
+        if (IssuesRecorder.NO_REFERENCE_BUILD.equals(referenceBuildId)) {
+            this.referenceBuildId = StringUtils.EMPTY;
+        }
+        else {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [File contains tab characters (this is the first instance).](https://app.codacy.com/manual/uhafner/warnings-ng-plugin/pullRequest?prid=4335332)